CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Fixes:
 Misc:
 - [#1734](https://github.com/rails-api/active_model_serializers/pull/1734) Adds documentation for conditional attribute (@lambda2)
 - [#1685](https://github.com/rails-api/active_model_serializers/pull/1685) Replace `IncludeTree` with `IncludeDirective` from the jsonapi gem.
+- [#1808](https://github.com/rails-api/active_model_serializers/pull/1808) Adds documentation for `fields` option. (@luizkowalski)
```

Comment:
The latest master has a [v0.10.1](https://github.com/rails-api/active_model_serializers/blob/master/CHANGELOG.md#v0101-2016-06-16) section. The version of the CHANGELOG in your PR has the newest release section as `v0.10.0 (2016-05-17)`. I believe other maintainers are asking that you rebase from rails-api/active_model_serializers master to get the latest CHANGELOG and ensure your CHANGELOG entry remains under the `master (unreleased)` section.